features/tutorial.feature

```diff
@@ -134,8 +134,8 @@ Feature: Following the tutorial
           - name: 'mongo'
             version: '3.4.0'
             config:
-              volumes:
-                - '{{EXO_DATA_PATH}}:/data/db'
+              named-volumes:
+                mongo: '/data/db'
```

Comment:
So I though we were originally going to be able to do this with `volumes_from` and not bother the user to define these in their service.yml. But it looks like docker-compose deprecated this functionality. So if we're going to make the users manually define volumes, we might as well just name this `volumes` and pass that straight through to docker-compose and use the standard docker syntax:
```yaml
volumes:
  - 'mongo:/data/db'
```

Reply:
In docker-compose, named docker volumes must also be listed in a separate volumes section at the bottom. We could parse the volumes the user passes in but I want to avoid having to write the code to determine if the volume is a name or a file path.

Reply:
Ugh that's obnoxious

Reply:
Can we at that point just use procedurally generated names for these and just call it `persist` or something?
```yaml
persist:
  - /data/db
```

Reply:
sounds good.

Reply:
Updated with the volume being named `<DEPENDENCY_NAME>_<PATH>` where path is converted to snake case.